abbr/views.py

```diff
@@ -13,10 +13,13 @@
     validate_url,
 )
 
+from ago import human
```

Comment:
In Python, third-party libraries commonly go below standard library imports and above local imports.